src/config/terminus_config.pl

```diff
@@ -109,6 +110,9 @@
     ->  false
     ;   Endpoint = Value).
 
+jwt_subject_claim_name(Name) :-
+    getenv_default('TERMINUSDB_JWT_SUBJ_CLAIM_NAME', 'http://terminusdb.com/schema/system#agent_name', Name).
```

Comment:
Is this a good standard? It is what we do for TerminusX but since it's non-standard maybe it's better to default to something that is normally in a JWT already?